recipes/libprotobuf/build.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+
+if [ "$(uname)" == "Darwin" ];
+then
+    # Switch to clang with C++11 ASAP.
+    export MACOSX_VERSION_MIN=10.7
+    export CC=clang
+    export CXX=clang++
+    export CXXFLAGS="-mmacosx-version-min=${MACOSX_VERSION_MIN}"
+    export CXXFLAGS="${CXXFLAGS} -stdlib=libc++ -std=c++11 -O2"
```

Comment:
Also same story with the `-O2` flag. This should move out of these `if` statements.